config/production.yaml.example

```diff
@@ -293,6 +293,30 @@ tracker:
   private: true
   # Reject peers that do a lot of announces (could improve privacy of TCP/UDP peers)
   reject_too_many_announces: false
+  
+customize_torrents:
+  # If additional trackers is more than 0, 
+  # then as many as additional trackers entries from the trakcers list will be 
+  # added to the torrent file
+  # entries are randomly chosen if number is less than whole list otherwise all
+  # 0 , empty or absense of this property means disabled   
+  # additional_trackers: 6
+  # trackers_list:
```

Comment:
Prefer to add `additional_trackers` array in https://github.com/Chocobozzz/PeerTube/blob/develop/config/production.yaml.example#L299 object